tests/test_pricer.py

```diff
@@ -132,15 +132,15 @@ def test_cuttingstock():
         rollUsage = 0
         solValue = round(s.getVal(pricer.data['var'][i]))
         if solValue > 0:
-            outline = 'Pattern_' + str(i) + ':\t' + str(solValue) + '\t\tCuts:\t'
+            outline = 'Pattern_' + str(i) + ':\t' + str(solValue) + '\t\tCuts:\t '
             for j in range(len(widths)):
                 rollUsage += pricer.data['patterns'][i][j]*widths[j]
                 widthOutput[j] += pricer.data['patterns'][i][j]*solValue
                 outline += str(pricer.data['patterns'][i][j]) + '\t'
             outline += 'Usage:' + str(rollUsage)
             print(outline)
 
-    print('\t\t\tTotal Output:','\t'.join(str(e) for e in widthOutput))
+    print('\t\t\tTotal Output:\t', '\t'.join(str(e) for e in widthOutput))
```

Comment:
The test is still not testing anything. Could you add some asserts of what is expected?

Reply:
Yes, good idea! Will do that.